web-app/js/portal/utils/trackUsage.js

```diff
@@ -6,6 +6,9 @@ function trackUsage(category, action, label, collection, value) {
             dimension1: collection
         });
     }
+    else {
+        //console.log(String.format("ga('send', 'event', category={0}, action={1}, label={2}, value={3})",category, action, label, collection, value));
```

Comment:
A wild console log appears

Reply:
@anguss00 I am now officially sick or re-writing this line every time that there is some work regarding tracking. I imagine if this line was available to be uncommented this bug would not even have come into existence.

Reply:
Fairo